### DISCUSSION TITLE
Console messages

### DISCUSSION THREAD

**Discussion by ssynowiec** [add2cal/add-to-calendar-button#523] [Q&A] [answered]
![image](https://github.com/add2cal/add-to-calendar-button/assets/52856724/fb5006bb-e7cc-4c73-b8e3-817cf604874a)
Can i remove this messages in console.log?

**Comment by jekuer** [comment, marked as answer]
Unfortunately not.
**This attribution and copyright notice is necessary to keep this project alive.**
Thanks for your understanding!
